_docs/dev-setup.md

```diff
@@ -1,5 +1,28 @@
 ---
 title: "Dev Setup: In-Depth Walkthrough"
+sidebar-title: "Dev Setup"
```

Comment:
I wish there was a better way of doing this, but from a glance at the Jekyll docs, it doesn't appear that there is an easy way to get all the headers from a file without writing a script to do so. Any ideas?

Reply:
Think you'd have to delve deeper into Ruby/Jekyll/Nokogiri? https://www.honeybadger.io/blog/automatically-generating-subnavigation-from-headings-in-jekyll/